src/components/ebay-snackbar-dialog/index.marko

```diff
@@ -0,0 +1,18 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+<ebay-dialog-base
+    ...input
+    isModal=false
+    classPrefix="snackbar-dialog"
+    open=state.open
+    class=[input.class, "snackbar-dialog--transition"]
+    buttonPosition="hidden">
+    <${input.renderBody}/>
+    <if(input.type === 'action')>
+    <@actions>
+        <span class="snackbar-dialog__actions">
+            <button on-click('emit', 'close') accesskey="u" class="fake-link snackbar-dialog__cta">Undo<span class="clipped"> - Access Key: U</span></button>
```

Comment:
This should be adjustable by user imo, like the text and accesskey.
I would add a `<@undo>` tag where the user can add what they need.
`<@undo accesskey="u" a11y-text="- access key: U">Undo</@undo>`
(Or split it in multiple attributes on the snackbar, like `undo-text` and `a11yUndoText`)
And then
```
<button on-click() ...processHTMLAttributes(input.undo, ['list of ignored stuff like a11yText'] >
  <${input.undo}/>
  <span class="clipped">${input.undo.a11yText}</span>
</button>
```

Remember there are non english usecases too!

Reply:
It's not always an undo. Just bear that in mind.

Reply:
Okay so I know you suggested the @undo tag, but I'm wondering whether just giving the user the `@action` tag and letting them put in the button as well as the text inside of it would be an adequate solution. 

Can you take a look at the change I just made and see whether it works?

Reply:
Yes action is probably better if it can be any action